vital/types/geo_point.cxx

```diff
@@ -1,5 +1,5 @@
 /*ckwg +29
- * Copyright 2017 by Kitware, Inc.
+ * Copyright 2019 by Kitware, Inc.
```

Comment:
Pedantic: KWIVER seems to like "2017-2019" or "2017, 2019". (Personally I think that's nuts, and we should *at least* just change the year, if not go to a banner that doesn't even include the year, but...)